### ISSUE TITLE
Support css files.

### ISSUE THREAD

**Issue by skorunka** [tinesoft/generator-ngx-library#131]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] feature request
```

### OS and Version?
Win 10

### Generator Version
5.1.0

### Tools Version
Output from: 2.0.0,
Output from: 8.7.0,
Output from: 5.5.1 npm

### Targeted Angular version
5

### Repro steps.
Use `css` file instead of `scss` in component `styleUrls`:

```
@Component({
	selector: 'eyc-grid-filter-column',
	templateUrl: './gridColumnFilter.component.html',
	styleUrls: ['./gridColumnFilter.component.scss']
})
export class GridColumnFilterComponent implements OnInit, AfterViewInit {
...
```

### The log given by the failure.
PS C:\temp\yo--ngx-library> gulp build
[22:18:52] Using gulpfile C:\temp\yo--ngx-library\gulpfile.js
[22:18:52] Starting 'clean:dist'...
[22:18:52] Starting 'clean:coverage'...
[22:18:52] Starting 'clean:build'...
[22:18:52] Finished 'clean:coverage' after 147 ms
[22:18:52] Finished 'clean:dist' after 156 ms
[22:18:52] Finished 'clean:build' after 171 ms
[22:18:52] Starting 'clean'...
[22:18:52] Finished 'clean' after 63 μs
[22:18:52] Starting 'build'...
[22:18:52] Starting 'compile'...
[22:18:52] Starting 'lint'...
[22:18:58] Finished 'lint' after 6.28 s
[22:18:58] Starting 'pre-compile'...
[22:18:58] Finished 'pre-compile' after 121 ms
[22:18:58] Starting 'inline-templates'...
PS C:\temp\yo--ngx-library>

### Desired functionality.
Support `.css` files.

**Comment by skorunka** [comment]
@tinesoft Why closed? Did I miss something?

**Comment by tinesoft** [comment]
Closed because it has been implemented!  :) see https://github.com/tinesoft/generator-ngx-library/commit/cbc54f9aaf1c850fea69909b9adb04d66236fbbf

**Comment by skorunka** [comment]
Oh, great! Thank you.

**Comment by tinesoft** [comment]
Just released [v5.2.0](https://github.com/tinesoft/generator-ngx-library/releases/tag/v5.2.0) with that feature. Enjoy!

**Comment by skorunka** [comment]
Well done! Thank you.
